mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactPage.js

```diff
@@ -15,6 +16,7 @@ import ShowArtifactTextView from './ShowArtifactTextView';
 import ShowArtifactMapView from './ShowArtifactMapView';
 import ShowArtifactHtmlView from './ShowArtifactHtmlView';
 import ShowArtifactPdfView from './ShowArtifactPdfView';
+import ShowArtifactTableView from './ShowArtifactTableView';
```

Comment:
Is it possible to lazily import this component here (using `React.lazy`)? This should end up having `papaparse` dependency in the separate chunk, keeping the main bundle clean and thin ✨